tests/phpunit/api/v4/Action/ContactGetTest.php

```diff
@@ -60,6 +62,138 @@ public function testGetDeletedContacts(): void {
     $this->assertContains($del['id'], $contacts->column('id'));
   }
 
+  /**
+   * Test ordering of contact by contact_sub_type when field
+   * contains more than one type and contact_type label and name
+   * are different enough to get sorted in different order.
+   *
+   *
+   */
+  public function testGetWithOrderBy(): void {
+ 
+    // Test dataset #1 : Contact sub type name, label
+    $cntctType = [['Membre', '1.Actif'],
+                  ['Proche', '2.Proche'],
+                  ['Soutien', '3.Soutien'],
+                  ['Employ_', '5.Employé'],
+                  ['DCD', '0.Décédé']
+    ];
```

Comment:
@syxys1 I'm not sure what you are basing your code style on. There are thousands of examples to follow in our test files and they all use the same consistent style. Can you please check other examples and base your style on that?
Our standard is 2-space indents and new arrays on a new line, and a comma on the last line, so this should be formatted like:
```suggestion
    $cntctType = [
      ['Membre', '1.Actif'],
      ['Proche', '2.Proche'],
      ['Soutien', '3.Soutien'],
      ['Employ_', '5.Employé'],
      ['DCD', '0.Décédé'],
    ];
```